### ISSUE TITLE
Experimental Flags not working

### ISSUE THREAD

**Issue by JoernBerkefeld** [Compass/compass#1449]
I'm working on Sass 3.3.0.rc.1 and Compass 0.13.alpha.10 and for some reason $experimental-support-for-svg and $support-for-original-webkit-gradients won't have any affect whatsoever. Neither SVGs nor the old gradient line appears in my CSS files. No errors are emitted.

My global.scss starts like this:
@charset "utf-8";

@import "compass";
$experimental-support-for-svg: true;
$support-for-original-webkit-gradients : true;

--- and then a bunch of partial imports.

I run compass via:
  sass --compass --sourcemap --watch scss:css
--> not via "compass watch" because of sourcemaps / Chrome debugging.

Am I doing something wrong or is this a bug? I just started using Sass & Compass today :/

**Comment by chriseppstein** [comment]
Jumping straight to the preview releases when you're trying to get started, eh? You're very brave.

The compass prefixing system has been completely overhauled and it can't be made backwards compatible.

Configuration variables should be set [before importing compass](http://compass-style.org/help/tutorials/configurable-variables/). If you had done so, you should have gotten warnings about this.

The new system is documented here on the beta documentation: http://beta.compass-style.org/help/documentation/tuning-vendor-prefixes/

**Comment by JoernBerkefeld** [comment]
thanks for the fast help - I did not find a link to the beta-docs (might be something to improve ;-)
I changed $svg-gradient-shim-threshold to 0.1 and the next build included SVGs - finally =D
BUT: The console shows this error: "Cannot determine the opposite position of: to"
The result is that it seems to default back to "left" / "to right" / a gradient from left to right for the SVG.

I use the current CSS3-"to DIRECTION"-syntax and as soon as disable SVG again, the error disappears. I encountered the same error when I tested "compass-sourcemap 12.4" and assume the SVG hack does not use the the new function

It does work if I use the old syntax instead.
